Gopkg.toml

```diff
@@ -9,7 +9,7 @@
 
 [[constraint]]
   name = "github.com/gin-gonic/gin"
-  version = "1.2.0"
+  branch = "master"
```

Comment:
Do we need to switch to `master` branch?

Reply:
yes, as the added dependency (bak-vaults) depends on the master

Reply:
The issue is that gin hasn't tagged a release a long time.